cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java

```diff
@@ -856,8 +860,8 @@ public void testBuildEmptyPackageWithCompilerPlugin() throws IOException {
         }
     }
 
-    @Test(description = "Build a ballerina project with the flag dump-graph")
-    public void testBuildBalProjectWithDumpGraphFlag() throws IOException {
+    @Test(dataProvider = "optimizeDependencyCompilation")
```

Comment:
Let's add the test description to the test.

Reply:
The description was removed since the test function name is descriptive enough.